client/src/html/customer_dashboard.html

```diff
@@ -0,0 +1,351 @@
+<!doctype html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link  rel="stylesheet" href="../css/lib/bootstrap.min.css" type="text/css">
+    <link  rel="stylesheet" href="../css/common.css" type="text/css">
+
+    <script type="text/javascript" src="../js/lib/jquery-1.11.3.min.js"></script>
+    <script type="text/javascript" src="../js/lib/jquery-ui.min.js"></script>
+    <script type="text/javascript" src="../js/lib/bootstrap.min.js"></script>
+    <script type="text/javascript" src="../js/lib/bootstrap-validator.js"></script>
+    <script type="text/javascript" src="../js/customer.js"></script>
+
+    <!-- angular -->
+    <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular.js"></script>
+    <script type="text/javascript" src="../js/controller/customer_controller.js"></script>
+
+    <script>
+        
+        $(document).ready(function(){
+            $( '#editMenu' ).click();
+            $( '#createMenu' ).click();
+
+            $(document).on("click","#generate_confirmation",function(){
+                generateConfirmation();
+            });
+        });
+
+    </script>
+</head>
+
+<body ng-app="RRS" ng-controller="customerController">

```

Comment:
why is the entire app in one container controller? Each smaller view should have their own controller